app/components/address_component.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AddressComponent < ViewComponent::Base
+  erb_template <<~ERB
```

Comment:
experiment - is this ok or does it cause more problems than its worth?

Reply:
I would say more problems than it is worth; having the template be separate seems cleaner. Also in case the component gets bigger, it wouldn't be as easy to read with template code in it.

Reply:
i was thinking about this when i was trying to sleep lol. i think one complaint we have deffo had with rails is theres too many files involved in making a change - like having these in 2 diff places is a bit infuriating! having it all in one file makes it super easy to edit! like i agree its a bit of a faff if it gets bigger but what if it doesnt? maybe gonna sleep on it a bit more lol